presto-product-tests/bin/product-tests-suite-4.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
this is very focused test suite. Maybe let's call it `product-tests-suite-tpcds`?